### PR TITLE
Fix integrity check error when owncloud is not installed

### PR DESCRIPTION
**Pull request by VicDeo**
## Description
- Fixes error for `php occ integrity:check-core` when ownCloud is not installed
- Force check all apps  when ownCloud is not installed

## Related Issue
```
> php occ integr:check-core -vvv
ownCloud is not installed - only a limited number of commands are available

                                                           
  [Symfony\Component\Debug\Exception\FatalThrowableError]  
  Call to a member function deleteAppValue() on null       
                                                           

Exception trace:
 () at owncloud/lib/private/IntegrityCheck/Checker.php:479
 OC\IntegrityCheck\Checker->cleanResults() at owncloud/lib/private/IntegrityCheck/Checker.php:595
 OC\IntegrityCheck\Checker->runInstanceVerification() at owncloud/core/Command/Integrity/CheckCore.php:60                                                                                                                                
 OC\Core\Command\Integrity\CheckCore->execute() at owncloud/lib/composer/symfony/console/Command/Command.php:262                                                                                                                         
 Symfony\Component\Console\Command\Command->run() at owncloud/core/Command/Base.php:159
 OC\Core\Command\Base->run() at owncloud/lib/composer/symfony/console/Application.php:826
 Symfony\Component\Console\Application->doRunCommand() at owncloud/lib/composer/symfony/console/Application.php:189                                                                                                                      
 Symfony\Component\Console\Application->doRun() at owncloud/lib/composer/symfony/console/Application.php:120                                                                                                                             
 Symfony\Component\Console\Application->run() at owncloud/lib/private/Console/Application.php:162
 OC\Console\Application->run() at owncloud/console.php:106
 require_once() at owncloud/occ:11

integrity:check-core [--output [OUTPUT]]

```

## Motivation and Context
Broken integrity check when owncloud is not installed

## How Has This Been Tested?
1. Download daily 
2. Extract daily
3. Do **not** install it
4. `php occ integrity:check-core`

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by PVince81** [comment]
Ah, I thought this was about signing without installing.

Still, a great step forward!

File: lib/private/IntegrityCheck/Checker.php [hunk 1]
```diff
@@ -594,10 +633,13 @@ public function verifyCoreSignature() {
 	public function runInstanceVerification() {
 		$this->cleanResults();
 		$this->verifyCoreSignature();
+
+		// FIXME: appManager === null means ownCloud is not installed. We check all apps in this case
```
Comment (PVince81) [inline lib/private/IntegrityCheck/Checker.php]: we could make that a documented behavior: when not installed it checks all apps

**Review by PVince81 — APPROVED** [review]
:+1: code looks fine

Did you retest all combinations ? (installed vs non-installed, scan app vs scan core in CLI and web UI?)

**Comment by DeepDiver1975** [comment]
> Did you retest all combinations ? (installed vs non-installed, scan app vs scan core in CLI and web UI?)

@VicDeo please answer and we can merge - THX

**Comment by VicDeo** [comment]
@DeepDiver1975 well, the answer for `scan app vs scan core in CLI and web UI` part is "no".

**Review by DeepDiver1975 — DISMISSED** [review]
@DeepDiver1975 well, the answer for scan app vs scan core in CLI and web UI part is "no".

**Comment by VicDeo** [comment]
### not installed
**makes no sense**  Web && core
**makes no sense**  Web && app
:white_check_mark:  CLI && core
:white_check_mark:  CLI && app

### installed
:white_check_mark:  CLI && core
:white_check_mark:  CLI && app 
:white_check_mark:   Web && core
**not implemented**  Web && app

**Comment by PVince81** [comment]
Thanks 👍

**Comment by PVince81** [comment]
Please backport for 10.0.5

**Comment by VicDeo** [comment]
Stable10: https://github.com/owncloud/core/pull/29692

**Pull request merged**